libseccomp-sys/src/lib.rs

```diff
@@ -3,22 +3,641 @@
 // Copyright 2021 Sony Group Corporation
 //
 
-#![allow(dead_code)]
-#![allow(non_upper_case_globals)]
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
-#![allow(clippy::all)]
 
-/// Manually support SCMP_ACT_ERRNO and SCMP_ACT_TRACE macro
-/// because the bindgen cannot expand the macros correctly.
+//! Raw FFI bindings for libseccomp library
+
+use std::os::raw::*;
+
+pub const SECCOMP_MODE_DISABLED: u64 = 0;
```

Comment:
Why did you change them from `u32` to `u64`?

Reply:
`#define` constants doesn't have a type, I then usually look at the function where those constants are used which is `prctl` in this case. `prctl` takes them as `arg2` which is an `unsigned long`.

Reply:
Great! I understand. Thank you